indico/modules/events/management/forms.py

```diff
@@ -228,28 +231,22 @@ class EventProtectionForm(IndicoForm):
                                             protected_object=lambda form: form.protected_object,
                                             acl_message_url=lambda form: url_for('event_management.acl_message',
                                                                                  form.protected_object))
-    acl = AccessControlListField(_('Access control list'), groups=True, allow_emails=True, allow_networks=True,
-                                 allow_external=True, default_text=_('Restrict access to this event'),
-                                 description=_('List of users allowed to access the event.'))
     access_key = IndicoPasswordField(_('Access key'), toggle=True,
                                      description=_('It is more secure to use only the ACL and not set an access key. '
                                                    '<strong>It will have no effect if the event is not '
                                                    'protected</strong>'))
     own_no_access_contact = StringField(_('No access contact'),
                                         description=_('Contact information shown when someone lacks access to the '
                                                       'event'))
-    managers = PrincipalListField(_('Managers'), groups=True, allow_emails=True, allow_external=True,
-                                  description=_('List of users allowed to modify the event'))
-    submitters = PrincipalListField(_('Submitters'), groups=True, allow_emails=True, allow_external=True,
-                                    description=_('List of users with submission rights'))
     visibility = SelectField(_("Visibility"), [Optional()], coerce=lambda x: None if x == '' else int(x),
                              description=_("""From which point in the category tree this event will be visible from """
                                            """(number of categories upwards). Applies to "Today's events" and """
                                            """Calendar only. If the event is moved, this number will be preserved."""))
+    permissions = PermissionsField(_("Permissions"))
```

Comment:
I would put this higher up so the field order here matches the order in which we show them in the form.